src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -92,3 +93,120 @@ message RefundResponse {
     REFUNDED = 3;
   }
 }
+
+// requires X-API-Key header with 'API Key' value
+message AuthorizationRequest {
+  string store_id = 1;                    // UUID of store
+  string amount = 2;                      // Amount in BigDecimal/2.dp (0.00)
+  string terminal_id = 3;                 // Send the payment to this terminal serial number
+  optional string trans_token = 4;        // Optional, max 60 chars
```

Comment:
What would be in the `trans_token` ?

Reply:
These card-type fields are all in the request? So it implies that they have been captured in the POS ?

Reply:
But we capture these in the terminal _normally_

Reply:
Or does this cater for manually entered card details? Again, something the terminal does...